app/src/pages/how-it-works.tsx

```diff
@@ -6,12 +6,15 @@ import Alert from '@components/Alert'
 import BackLink from '@components/BackLink'
 import ButtonLink from '@components/ButtonLink'
 
-const HowItWorks: NextPage = () => {
+import { Page } from '@src/types'
+
+const HowItWorks: NextPage<Page> = (props: Page) => {
+  const { backRoute = '', forwardRoute = '' } = props
   const listCopyKeys: string[] = ['apply', 'eligible', 'appointment']
 
   return (
     <>
-      <BackLink href="/" />
+      <BackLink href={backRoute} />
```

Comment:
Am I understanding correctly that a set of changes in this PR is around instead of hardcoding a route passing it through as a prop so it can be responsive or change based on the situation / business logic?

Reply:
Correct! I wanted to centralize the routing and business logic for 2 main reasons:

1. A page is only responsible for knowing how to internally manage itself. It doesn't need to know about other pages and how it relates to them
2. Keeping all the business logic/routing in one place makes it easier to see at a high level how all the pages interact and makes it easier to make updates in the future